README.md

```diff
@@ -138,6 +138,44 @@ quantized_model = nncf.quantize(model, calibration_dataset)
 
 </details>
 
+<details><summary><b>TorchFX</b></summary>
+
+```python
+import nncf
+import torch.fx
+from torchvision import datasets, models
+from torch._export import capture_pre_autograd_graph
+from nncf.torch import disable_patching
+
+# Instantiate your uncompressed model
+model = models.mobilenet_v2()
+
+# Provide validation part of the dataset to collect statistics needed for the compression algorithm
+val_dataset = datasets.ImageFolder("/path", transform=transforms.Compose([transforms.ToTensor()]))
+dataset_loader = torch.utils.data.DataLoader(val_dataset)
+
+# Step 1: Initialize the transformation function
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+# Step 2: Initialize NNCF Dataset
+calibration_dataset = nncf.Dataset(dataset_loader, transform_fn)
+
+# Step 3: Export model to TorchFX
+input_shape = (1, 3, 224, 224)
+with nncf.torch.disable_patching():
+    # Using capture_pre_autograd_graph to export torch.fx.GraphModule
+    # the same way it done for PyTorch 2 Export Post Training Quantization:
+    # https://pytorch.org/tutorials/prototype/pt2e_quant_ptq.html
+    fx_model = capture_pre_autograd_graph(model.eval(), args=torch.ones(input_shape))
```

Comment:
Torch 2.5.0 introduced `export_for_training` method which is used for quantization pipeline https://pytorch.org/executorch/stable/tutorials/export-to-executorch-tutorial.html I'm waiting that NNCF will support torch 2.5.0 in the upcoming release. I would suggest to merge this PR after clarification the status of the task of supporting torch 2.5.0 in NNCF.
@anzr299, @MaximProshin

Reply:
Done, please check